.changeset/neat-kiwis-know.md

```diff
@@ -0,0 +1,18 @@
+---
+'@leafygreen-ui/password-input': major
+---
+
+[LG-4135](https://jira.mongodb.org/browse/LG-4135)
+
+1. Updated styling:
+  - updated spacing for `'small'` and `'large'` size variants
+  - added `'xsmall'` variant
+  - updated placeholder text color
+  - removed validation icon from inside the input
+
+2. Added `errorMessage` and `successMessage` props
+  - `errorMessage` will render a default of `'This input needs your attention'` and can be customized
```

Comment:
![Screenshot 2024-04-10 at 1 09 09 PM](https://github.com/mongodb/leafygreen-ui/assets/5740159/258035dd-9c20-4d24-bdcc-1accf51617da)
the custom container story is missing the icon and default message, is that correct?

Reply:
yes, this is intentional if a custom error is defined outside of the component. I've updated the story to include this

Reply:
i'm curious, why we don't want to show the icon anymore?